{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/events/models.py

```diff
@@ -88,5 +99,27 @@ def date(self):
 
         return date_string
 
+    def schema(self):
+        schema = {
+            '@context': 'http://schema.org',
+            '@type': 'Event',
+            'startDate': self.start_date,
+            'endDate': self.end_date,
+            'description': self.summary if self.summary else '',
+            'name': self.title,
+            'mainEntityOfPage': 'https://www.{}{}'.format(settings.SITE_DOMAIN, self.get_absolute_url()),
```

Comment:
Use `url_from_path` from `utils/utils.py` for optimum results - this handles cases like no HTTPS (for local development) and doesn't assume a `www.` prefix.